db/schema.rb

```diff
@@ -141,7 +141,7 @@
     t.integer  "client_data_id"
     t.string   "client_data_type", limit: 255
     t.integer  "requester_id"
-    t.string   "public_id"
+    t.string   "public_id",        limit: 255
```

Comment:
git checkout since this is unrelated to this PR? (running migrations also changes this for me -- not sure why it keeps going back and forth